rocky/assets/css/components/plugins.scss

```diff
@@ -35,10 +36,15 @@ $normalizer-bg: #e17000;
 
     /* Checkbox */
     >input[type="checkbox"] {
+      accent-color: var(--form-accent-color-color, initial);
+      cursor: var(--form-checkbox-cursor);
       left: 1rem;
-      margin-top: 0;
+      margin: 0;
       position: absolute;
       top: 1rem;
+      width: var(--form-checkbox-width);
+      min-height: var(--form-checkbox-width);
+      height: var(--form-checkbox-width);
```

Comment:
This also changes non reporting parts where this is used, are these changes also correct for those parts? I don't know enough to understand that, but I want to make sure we don't introduce any regressions when we merge this as preview.

Reply:
Yes I made sure that this is also reflected at other parts using especially the plugin tiles. I will make later another PR to make the tiles a bit more generic as we now also introduce different style of tiles, now also having the report types tile.